packages/appkit-utils/src/CaipNetworkUtil.ts

```diff
@@ -109,10 +110,18 @@ export const CaipNetworksUtil = {
     return `${ConstantsUtil.CHAIN.EVM}:${network.id}` as CaipNetworkId
   },
 
-  getRpcUrl(caipNetwork: AppKitNetwork, caipNetworkId: CaipNetworkId, projectId: string) {
+  // eslint-disable-next-line max-params
+  getRpcUrl(
+    caipNetwork: AppKitNetwork,
```

Comment:
I'd change this for `defaultRpcUrl` so it's more clear. 
Otherwise the `caipNetwrok`rpc is going to be overwritten